spark/src/main/kotlin/com/adevinta/spark/components/chips/Chip.kt

```diff
@@ -235,6 +241,9 @@ public fun Chip(
                         color = contentColor,
                     )
                 }
+                // Fix to prevent chip from having a width of zero when used
+//                // in a scroll container
+                .widthIn(max = 9000.dp)
                 .defaultMinSize(minHeight = ChipDefaults.MinHeight)
```

Comment:
Can you cleanup this comment? (remove the duplicated `//`)

Reply:
Also, IMO this should probably be extracted as an extension function to be re-used/documented.

Reply:
Will do but this is not something I want to [stay in the codebase](https://github.com/adevinta/spark-android/issues/1238) as this is the quick fix for a hack in Polaris to allow trailing icon in chips.

Reply:
Done in [fe8c41c](https://github.com/adevinta/spark-android/pull/1236/commits/fe8c41c37836b6173f6030041f5959f7bfb6ab08)

Reply:
@soulcramer I meant to extract it like this

```kotlin
/** Workaround for bug XYZ */
private fun Modifier.fixChipsWhatever() = widthIn(max = 9000.dp)
```